src/js/main.js

```diff
@@ -42,6 +42,24 @@ export function init(el, context, config) {
             hideAboutBtn.classList.remove('docs--show-about');
         };
 
+
+        // Show the trailer on click #docs__playTrailer
+        const $showTrailerAll = builder.querySelectorAll('.docs__shows-trailer');
```

Comment:
we don't prefix variables with `$` elsewhere - we probably want to agree on a variable name style and be consistent across the project